src/gui/MainWindow.cpp

```diff
@@ -64,6 +65,21 @@
 
 #include "lmmsversion.h"
 
+#if !defined(LMMS_BUILD_WIN32) && !defined(LMMS_BULID_APPLE) && !defined(LMMS_BUILD_HAIKU)
```

Comment:
Is this check still needed? Since @BumblingCoder switched the loader to `QLibrary`, it can be removed safely. If there are some reason to keep this, however, leaving is okay, too.

Reply:
It should be safe to run the code on all platforms. Pointless on the ones it is disabled on, but safe.

Reply:
It saves a few cpu cycles to preprocess it out.  I agree with the logic, especially when loading libraries.  Why add an unnecessary step to MainWindow for systems which can't experience the bug?

Reply:
> Why add an unnecessary step to MainWindow for systems which can't experience the bug?

Okay then. Keep it if leaving makes sense.